annotation-client/src/test/resources/subtitle/minimal.json

```diff
@@ -3,8 +3,8 @@
   "body": {
     "type": "FullTextResource",
     "language": "it",
-    "value": " WEBVTT\nNOTE Paragraph\n00:00:00.988 --> 00:00:03.572\nDolo le ultime novità\ndella moda per signora\n...\n00:00:30.359 --> 00:00:33.608\ncon il grande finale:\nCarnevale con il popolo eschimese.\n\n00:00:34.199 --> 00:00:35.780\n <special char/> musica) <special char/> ",
-    "format": "text/vtt"
+    "value": "deuhdiwuehdiwuehdwiuehdwieuhdew",
```

Comment:
this file is used is more integration tests, please revert back and create a new test file if needed

Reply:
Done.